frontend/ui/widget/focusmanager.lua

```diff
@@ -249,6 +252,15 @@ function FocusManager:onFocusMove(args)
     return true
 end
 
+function FocusManager:onPhysicalKeyboardConnected()
+    -- Re-initialize with new keys info. It's called without "self:" in case descendants override init.
+    FocusManager.init(self)
```

Comment:
I experimented with adding the `self:init()` call too. This re-initializes the menus and adds key shortcuts (letters within squares) to the file browser. But it's not clear if calling init twice is safe in the general case. Also, the interface for the file browser refreshes not immediately after connecting a keyboard. I have to open a book and quit, which creates an impression of inconsistent behavior.